nfp/preprocessing/xtb_preprocessor.py

```diff
@@ -0,0 +1,131 @@
+from typing import Dict, Optional, Callable, Hashable
+
+import networkx as nx
+import numpy as np
+import rdkit.Chem
+import json
+
+try:
+    import tensorflow as tf
+except ImportError:
+    tf = None
+
+from nfp.preprocessing import features
+from nfp.preprocessing.preprocessor import Preprocessor
+from nfp.preprocessing.tokenizer import Tokenizer
+
+class xTBPreprocessor(Preprocessor):
```

Comment:
Is there any reason you don't inherit from MolPreprocessor? Then you wouldn't have to redefine a lot of the same things. Is the idea that this is essentially the same as the MolPreprocessor class, but you want to be able to add some additional properties calculated via xTB?

Reply:
Done.